go.mod

```diff
@@ -71,8 +76,9 @@ require (
 	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
 	github.com/golang/snappy v0.0.4 // indirect
 	github.com/hashicorp/hcl v1.0.0 // indirect
-	github.com/hyperledger/fabric-amcl v0.0.0-20210603140002-2670f91851c8 // indirect
+	github.com/hyperledger/fabric-amcl v0.0.0-20230602173724-9e02669dceb2 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
+	github.com/kilic/bls12-381 v0.1.0 // indirect
```

Comment:
@ale-linux The validation plugin is failing due to this new kilic dependency that was added to IBM/Idemix, any ideas?

Reply:
building with `-tags=generic,purego` fixes the problem. It's linked to https://github.com/kilic/bls12-381/issues/42 and https://github.com/Consensys/gnark-crypto/issues/468.